WalletWasabi.Gui/Controls/WalletExplorer/SendTabView.xaml

```diff
@@ -33,10 +33,22 @@
             <TextBlock Text="{Binding WarningMessage}" Classes="warningMessage" />
             <TextBlock Text="{Binding ValidationMessage}" Classes="errorMessage" />
         </StackPanel>
-        <controls:GroupBox Title="{Binding Title}" TextBlock.FontSize="30" Padding="20">
-            <Grid Classes="content">
-                <local:CoinListView DataContext="{Binding CoinList}" />
-            </Grid>
+        <controls:GroupBox Title="{Binding Title}" TextBlock.FontSize="30" Padding="20" Margin="10">
+
+        <Grid Classes="content">
+            <DockPanel LastChildFill="True">
+                <Grid ColumnDefinitions="40,80,100,600,100,150,Auto" Margin="5 0" DockPanel.Dock="Top">
+                    <TextBlock Text="Select" />
+                    <TextBlock Text="Confirmed" Grid.Column="1"/>
+                    <TextBlock Text="Amount" Grid.Column="2" />
+                    <TextBlock Text="Label" Grid.Column="3" />
+                    <TextBlock Text="Privacy Level" Grid.Column="4" Margin="20 0 0 0"/>
+                    <TextBlock Text="" Grid.Column="5"/>
+                    <TextBlock Text="" Grid.Column="6"/>
+                </Grid>
+            <local:CoinListView DataContext="{Binding CoinList}" />
```

Comment:
might just be GitHub site, but <local:CoinListView should be indented to be inline with </Grid>

vs can format the code automatically unfortunately for you not possible :(